cabal.project

```diff
@@ -85,53 +85,9 @@ constraints:
   , protolude >= 0.3.2
 
   -- TODO: these should be set in cabal files, but avoiding setting them in lower dependencies for initial CHaP release
-  , cardano-prelude == 0.1.0.1
-  , base-deriving-via == 0.1.0.0
+
+  -- Needed until ouroboros-network gets proper bounds
   , cardano-binary  == 1.5.0
-  , cardano-binary-test == 1.3.0
-  , cardano-crypto-class  == 2.0.0.1
-  , cardano-crypto-praos  == 2.0.0.0.1
-  , cardano-crypto-tests  == 2.0.0.0.1
-  , cardano-slotting  == 0.1.0.0
-  , measures == 0.1.0.0
-  , orphans-deriving-via == 0.1.0.0
-  , cardano-strict-containers == 0.1.0.1
-  , plutus-core == 1.0.0.1
-  , plutus-ledger-api == 1.0.0.1
-  , plutus-tx == 1.0.0.0
-  , plutus-tx-plugin == 1.0.0.0
-  , prettyprinter-configurable == 0.1.0.0
-  , plutus-ghc-stub == 8.6.5
-  , word-array == 0.1.0.0
-  , io-classes >= 0.2.0.0
-  , typed-protocols == 0.1.0.1
-  , typed-protocols-examples == 0.1.0.1
-  , ekg-forward >= 0.2
-
-  , cardano-ledger-alonzo == 0.1.1.1
```

Comment:
to be clear, I think a bunch of these should turn into lower bounds, at least, rather than relying on "picking the latest". Good to specify at least the major lower bound.

Reply:
Probably pinning the major version is the thing to do.